config/clusters/nasa-veda/common.values.yaml

```
@@ -96,6 +96,38 @@ basehub:
                   slug: pangeo
                   kubespawner_override:
                     image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:5068290376e8c3151d97a36ae6485bb7ff79650b94aecc93ffb2ea1b42d76460
+                    init_containers:
+                      # Need to explicitly fix ownership here, as otherwise these directories will be owned
+                      # by root on most NFS filesystems - neither EFS nor Google Filestore support anonuid
+                      - name: volume-mount-ownership-fix
+                        image: busybox
+                        command:
+                          [
+                              "sh",
+                              "-c",
+                              "id && chown 1000:1000 /home/jovyan && chown 1000:1000 && ls -lhd /home/jovyan ",
+                          ]
```

This was initially just meant as a refactoring to cleanup this code section, but actually it ended up fixing a bug. Note how there is a segment saying `&& chown 1000:1000 &&`

```suggestion
                          - sh
                          - -c
                          - id && chown 1000:1000 /home/jovyan && ls -lhd /home/jovyan
```